src/State/Scope.php

```diff
@@ -298,8 +309,8 @@ public function applyToEvent(Event $event, int $maxBreadcrumbs = 100): ?Event
         $event->getExtraContext()->merge($this->extra->toArray());
         $event->getUserContext()->merge($this->user->toArray());
 
-        foreach ($this->eventProcessors as $processor) {
-            $event = $processor($event);
+        foreach (array_merge(self::$globalEventProcessors, $this->eventProcessors) as $processor) {
```

Comment:
Why do we have two of them?

Reply:
Because there are "global" and scope related EventProcessors.
Integrations use the "global" event processors but as an example if an user wants to change an event in the context of a scope, the can 
```
withScope(function($scope) {
  $scope->addEventProcessors(function ($event) {
      // Changing $event
     return null; // or dropping it
  });
});